static/js/util/api.js

```diff
@@ -62,6 +66,10 @@ function fetchJSONWithCSRF(input, init) {
     // Also note that text is a promise here, not a string
     let text = response.text();
 
+    // For 400 and 401 errors, force login
```

Comment:
Can you add a comment about the refresh token string for the 400? Otherwise I'm fine with this since we're only using it on getDashboard